protocol/go.mod

```diff
@@ -432,7 +432,7 @@ replace (
 	github.com/cometbft/cometbft => github.com/dydxprotocol/cometbft v0.38.6-0.20240426214049-c8beeeada40a
 	// Use dYdX fork of Cosmos SDK
 	github.com/cosmos/cosmos-sdk => github.com/dydxprotocol/cosmos-sdk v0.50.6-0.20240326192503-dd116391188d
-	github.com/cosmos/iavl => github.com/dydxprotocol/iavl v1.1.1-0.20240408175732-0fca9d69cbc4
+	github.com/cosmos/iavl => github.com/dydxprotocol/iavl v1.1.1-0.20240509160433-4cb59900bfb2
```

Comment:
Question on the underlying [commit](https://github.com/dydxprotocol/iavl/commit/4cb59900bfb27a7525185bc10e90c6b3bb487c12):   instead of allowing `roy/*` should we specify [roy/v1.x.x-fix](https://github.com/dydxprotocol/iavl/compare/roy/v1.x.x-fix) instead to not push future test commits

Reply:
Doesn't matter. Rule only applies for the branch itself.

Reply:
Ok, we don't plan to continue managing the our iavl fork right? We are just using this commit for the WorkingHash fix